### PR TITLE
Add created_by and modified_by fields in ScriptVersion

### PR DESCRIPTION
**Pull request by Lioscro**
* Minor fix for Celery>=5 where the `-A` option usage changed by setting celery<5
* Added `created_by` and `modified_by` fields for `ScriptVersion`
* Admin page now displays the `created` and `modified` fields for `ScriptVersion`. Currently, I have all of them set to be read-only, but still a bit unsure if this is the best, or should we make these editable?
* Display script version, iteration creator and modifier in UI script view.

image

### CODE REVIEW

File: wooey/models/core.py [hunk 1]
```diff
@@ -108,6 +108,13 @@ class ScriptVersion(ModelDiffMixin, WooeyPy2Mixin, models.Model):
     created_date = models.DateTimeField(auto_now_add=True)
     modified_date = models.DateTimeField(auto_now=True)
 
+    created_by = models.ForeignKey(
+        User, related_name='created_script_version', on_delete=models.SET_NULL, null=True, blank=True
```
Comment (Chris7) [inline wooey/models/core.py:112]: This should be plural, can we keep the convention of `set` and make these `created_script_version_set`?

File: setup.py [hunk 2]
```diff
@@ -15,7 +15,7 @@
     entry_points={'console_scripts': ['wooify = wooey.backend.command_line:bootstrap', ]},
     python_requires='>3.5.0',
     install_requires=[
-        'celery>=4.0,<6',
+        'celery>=4.0,<5',
```
Comment (Chris7) [inline setup.py:18]: Can you elaborate on the `-A` option? Might need to update docs and the bootstrap for this

Comment (Chris7) [reply setup.py:18]: Or separate this into another MR in case we do need that.

**Review by Chris7 — COMMENTED** [review]
Thanks for the changes! Some minor stuff

File: setup.py (hunk 2 above)
Comment (Chris7) [reply setup.py:18]: It looks like it's still valid, how are we calling it at ginkgo? https://docs.celeryq.dev/en/stable/django/first-steps-with-django.html#starting-the-worker-process

Comment (Lioscro) [reply setup.py:18]: It looks like the`-A` option moved to become a global option, starting 5.0
```
wooey-celery-1  | Error: No such option: -A
wooey-celery-1  | You are using `-A` as an option of the worker sub-command:
wooey-celery-1  | celery worker -A celeryapp <...>
wooey-celery-1  |
wooey-celery-1  | The support for this usage was removed in Celery 5.0. Instead you should use `-A` as a global option:
wooey-celery-1  | celery -A celeryapp worker <...>
```

Comment (Lioscro) [reply setup.py:18]: Is there a reason why we can't pin `celery>=5.0` and move `-A` to be global?

File: wooey/admin.py [hunk 3]
```diff
@@ -68,8 +69,14 @@ def script_versions(self, obj):
         return ', '.join(['{}: {}'.format(script_version.script.script_name, script_version.script_iteration) for script_version in obj.script_version.all()])
 
 class ScriptVersionAdmin(ModelAdmin):
-    list_display = ('script', 'script_version', 'script_iteration', 'default_version')
-
+    list_display = ('script', 'script_version', 'script_iteration', 'default_version', 'created_by', 'modified_by')
+    readonly_fields = ('created_date', 'created_by', 'modified_date', 'modified_by')
+
+    def save_model(self, request, obj, form, change):
+        if not change:
```
Comment (Chris7) [inline wooey/admin.py:76]: If this already exists won't we always update created by with the last user? Seems like this should be checking if created by is empty

File: wooey/templates/wooey/scripts/script_view.html [hunk 4]
```diff
@@ -14,6 +14,13 @@
     <div class="page-header">
         <h3 id="wooey-script-title">{{ script.script_name }}</h3>
         <p id="wooey-script-description">{{ script.script_description }}</p>
+
+        <p id="wooey-script-info" style="color:gray"><small>
```
Comment (Chris7) [inline wooey/templates/wooey/scripts/script_view.html:18]: This is ok, but I'll probably move it elsewhere. Seems like we're putting a lot of stuff above where users primarily interact with the script.

Comment (Chris7) [reply wooey/templates/wooey/scripts/script_view.html:18]: Also should handle the none case better than just saying none

File: wooey/admin.py (hunk 3 above)
Comment (Lioscro) [reply wooey/admin.py:76]: Nope! `change` is `True` only when updating an existing entry and `False` when creating a new one.
image

File: wooey/templates/wooey/scripts/script_view.html (hunk 4 above)
Comment (Lioscro) [reply wooey/templates/wooey/scripts/script_view.html:18]: Sounds good. Do you think somewhere a bit more discrete, like a footer?

Comment (Chris7) [reply wooey/templates/wooey/scripts/script_view.html:18]: I was thinking to put the information in the header, and only put the last modified/version stuff in. like:
image

And if the user is unknown -- omit it.

Comment (Lioscro) [reply wooey/templates/wooey/scripts/script_view.html:18]: image

**Comment by Chris7** [comment]
Great!

**Pull request merged**